sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/models/EncryptResult.java

```diff
@@ -9,33 +9,40 @@
  * Represents the details of encrypt operation result.
  */
 public final class EncryptResult {
```

Comment:
Should these model classes be annotated with `@Immutable`?

Reply:
updated, the annotation only enforces checkstyle checks.